src/cli/src/http.rs

```diff
@@ -26,17 +25,26 @@ pub async fn execute(request: Request) -> Result<Response, CliError> {
     let addr: (&str, u16) = (&host, request.url().port_or_known_default().unwrap_or(443));
     let tcp_stream = fluvio_future::net::TcpStream::connect(addr).await?;
     debug!("Established TCP stream");
-    let tls_connector = create_tls().await?;
+    let tls_connector = create_tls().await;
     debug!("Created TLS connector");
-    let tls_stream = tls_connector.connect(&host, tcp_stream).await?;
+    let tls_stream = tls_connector.connect(host, tcp_stream).await?;
     debug!("Opened TLS stream from TCP stream");
     let response = client::connect(tls_stream, request).await?;
 
     debug!(?response, "Http response:");
     Ok(response)
 }
+async fn create_tls() -> fluvio_future::native_tls::TlsConnector {
+    fluvio_future::native_tls::TlsConnector::default()
+}
 
-async fn create_tls(
-) -> Result<fluvio_future::openssl::TlsConnector, fluvio_future::openssl::TlsError> {
-    Ok(fluvio_future::openssl::TlsConnector::builder()?.build())
+#[fluvio_future::test_async]
```

Comment:
use `test` rather than `test_async`.  this way, it doesn't need to return result

Reply:
I mean, I kinda like using `Result` as it allows for the use of `?` in the unit test.

Reply:
Problem is that it's hard to trace if test fails.
so instead of:
```
  let index = agent.request_index()?;
```
should be like:
```
  let index = agent.request_index().expect("can't find index");
```

Reply:
Done.